README

```diff
@@ -51,6 +50,24 @@ Configure Delphes on lxplus.cern.ch
 
   make
 
+Install Delphes from conda-forge
```

Comment:
I'm a bit confused by this `README` file as they way it is currently edited it mixes Markdown (the use of the `conda-forge` URL link) and plain text (the code snippets). Does this get used somewhere else outside of the repository? If not, what is the point of this file?